blog.html

```diff
@@ -0,0 +1,93 @@
+    <title>Adopt A Dog</title>
+  </head>
+  <body>
+    <header>
+      <figure class="company-logo">
+        <img src="images/adoptadog-logo.svg" alt="Company Logo">
+      </figure>
+
+      <figure class="cart-icon">
+        <img src="images/cart-icon.svg" alt="Cart Icon">
+      </figure>
+
+      <nav>
+        <ul>
+          <li class="navlist"><a href="index.html">Home</a></li>
+          <li class="navlist"><a href="dogs.html">Dogs</a></li>
+          <li class="navlist"><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+
+    </header>
+    <main>
+      <h1>Adoptadog Blog</h1>
+
+      <article class="blog">
+
+        <figure>
+          <img src="images/blog-1.jpg" alt="Dog and owner hiking">
+        </figure>
+
+        <h2>Traveling With Your Dog</h2>
+
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+
+        <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+      </article>
+
+      <article class="blog">
+
+        <figure>
+          <img src="images/blog-2.jpg" alt="Dog walker walking four dogs">
+        </figure>
+
+        <h2>How To Walk Multiple Dogs</h2>
+
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+
+        <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+      </article>
+
+      <article class="blog">
+
+        <figure>
+          <img src="images/blog-3.jpg" alt="Owner playing fetch with dog">
+        </figure>
+
+        <h2>Teach Your Dog To Fetch!</h2>
+
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+
+        <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+      </article>
+
+
+    </main>
+
+    <footer>
+      <figure>
+        <img src="images/adoptadog-logo.svg" alt="Company Logo">
+      </figure>
+      <small>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</small>
```

Comment:
I'm not a huge fan of using the small element. It's fine to use, but I'd rather you wrap it in a span (or div) an style from there. Relying on the browser to render small right can be somewhat tricky. This is just my opinion and what you're doing isn't wrong, but I'd steer you towards a span